### PR TITLE
doc/user: add temporary submit results steps

### PR DESCRIPTION
**Pull request by mtulio**
User documentation to provide a way to send the results to Red Hat team.

Waiting for the definition of the initial steps.

https://issues.redhat.com/browse/SPLAT-724

### CODE REVIEW

**Comment by bostrt** [comment]
So far, this is looking good!

File: docs/user.md [hunk 1]
```diff
@@ -168,6 +169,37 @@ You can see a summarized view of the results using:
...
+To submit the results to the Red Hat to review the archive, you should patch by running the following script:
+
+```sh
+mkdir retrieved-archive-patched
+tar xfz retrieved-archive.tar.gz -C retrieved-archive-patched/
+find retrieved-archive-patched/ -iname packages.operators.coreos.com_v1_packagemanifests.json |xargs rm -rvf
+tar cfz retrieved-archive-patched.tar.gz retrieved-archive-patched/
+```
+
+Then, submit the `retrieved-archive-patched.tar.gz` to your Red Hat partner contact. -->
```
Comment (mtulio) [inline docs/user.md:202]:
```suggestion
```

File: docs/user.md [hunk 2]
```diff
@@ -168,6 +169,24 @@ You can see a summarized view of the results using:
...
+- Go to [`Support > My support tickets > Create Case`](https://connect.redhat.com/support/technology-partner/#/case/new).
+- In the `Request Category` step, select `Product Certification`.
+- In the `Product Selection` step, for Product field, select `OpenShift Container Platform` and select the Version you are using.
+- Click `Next` to continue.
+- In the `Request Details` step, in the `Request Summary` field, specify `[VCSP] OpenShift Provider Certification Tool Test Results` and provide any additional details in the `Please add description` field.
+- Click `Next` to continue.
+- Click `Submit` when you have complete all the required information.
+- A Product Certification case will be created, and please follow the instructions provided to add the test results and any other related material for us to review.
+- Go to [`Support > My support tickets`](https://connect.redhat.com/support/technology-partner/#/case/list) to find the case and review status and/or to add comments to the case.
+- Attach the `retrieved-archive.tar.gz` result file to the case.
```
Comment (mtulio) [inline docs/user.md:188]:
```suggestion
- Attach the `retrieved-archive.tar.gz` result file to the case.
- Attach the `must-gather.tar.gz` file to the case.
```

**Pull request merged**